Example/Tests/TypedDataObjectHashingTests.swift

```diff
@@ -0,0 +1,173 @@
+//
+// This source file is part of the 0x.swift open source project
+// Copyright 2019 The 0x.swift Authors
+// Licensed under Apache License v2.0
+//
+// TypedDataObjectHashingTests.swift
+//
+// Created by Igor Shmakov on 24/05/2019
+//
+
+import XCTest
+import Nimble
+import Quick
+import Web3Swift
+
+@testable import Swifty0x
+
+
+struct MockedEIP712Domain: EIP712Representable {
```

Comment:
Let's put mock objects in a separate folder from tests itself

Reply:
These objects are related to this test only